scripts/train/train.py

```diff
@@ -509,6 +510,8 @@ def main(cfg: DictConfig) -> Trainer:
         eval_loader.metric_names = eval_metric_names
         evaluators.insert(0, eval_loader)  # Put the base eval_loader first
 
+    logging.getLogger('llmfoundry').setLevel(python_log_level.upper())
```

Comment:
could you add to `eval.py` as well?

Reply:
python_log_level can be `None`, so lets just copy the way that composer does it (including the basicconfig setup)?
https://github.com/mosaicml/composer/blob/8d59da2293499f1faabff6276eee98344f334c64/composer/trainer/trainer.py#L902-L911